### PR TITLE
0.1.9

### PR DESCRIPTION
**Pull request by pq**
@devoncarew

### CODE REVIEW

**Comment by alexander-doroshko** [comment]
Sorry, I still didn't review `FlutterProjectOpenProcessor` thoroughly. Are you sure you want to keep `isStrongProjectInfoHolder()` overridden?

**Comment by pq** [comment]
Will fix this today: https://github.com/flutter/flutter-intellij/issues/684

File: resources/META-INF/plugin.xml [hunk 1]
```diff
@@ -21,6 +21,18 @@
 
   <change-notes>
     <![CDATA[
+0.1.9:
+<ul>
+  <li>preliminary windows support</li>
```
Comment (devoncarew) [inline resources/META-INF/plugin.xml]: let's back this out - there's still a bunch of work in flutter_tools

**Comment by devoncarew** [comment]
lgtm; can also add a line item about webstorm support

**Comment by pq** [comment]
> lgtm; can also add a line item about webstorm support

How about:

> initial webstorm IDE support

Or should we be more assertive?

File: resources/META-INF/plugin.xml (hunk 1 above)
Comment (pq) [inline resources/META-INF/plugin.xml]: Done.

**Pull request merged**